org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -1064,15 +1065,9 @@ else if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), functions[f]))
 										displayType = getDisplayType(value);
 									
 										if (display.startsWith("@SCRIPT")) {
-											display = display.replace("@SCRIPT", "");
-											value = parseVariable(display, pdc, pd);
-											Interpreter bsh = new Interpreter ();
-											try {
-												value = bsh.eval(value.toString());
-											}
-											catch (EvalError e) {
-												log.severe(e.getMessage());
-											}
+											value = display.replace("@SCRIPT", "");
```

Comment:
I think better just keep it as existing, i.e:
display = display.replace("@SCRIPT", "");

Reply:
We can remove this line since the `display` variable is not used after this and move the `.replace("@SCRIPT", "")` to the new parsing section